analysis/bin/massAnalyser.cc

```diff
@@ -121,6 +120,7 @@ void massAnalyser::collectTMVAvalues() {
     b_cme_pdgId = had_pdgId[i];
     b_cme_tmva_bdtg = bdtg->EvaluateMVA("BDTG");
     //b_cme_nMatched = hadTruth_nMatched[i];
+    if (abs(b_cme_pdgId) != 421) continue;
```

Comment:
Hi Dongjun. This should really be at the top of the loop. If its not D0, then really, theres no reason to fill out all the variables and evaluate the MVA, its just going to waste time when you run.